src/webpack/patchWebpack.ts

```diff
@@ -172,26 +172,36 @@ function patchFactories(factories: Record<string, (module: any, exports: any, re
 
             if (!exports) return;
 
-            // There are (at the time of writing) 11 modules exporting the window
-            // Make these non enumerable to improve webpack search performance
             if (require.c) {
-                let foundWindow = false;
-
-                if (exports === window) {
-                    foundWindow = true;
-                } else if (typeof exports === "object") {
-                    if (exports?.default === window) {
-                        foundWindow = true;
-                    } else {
-                        for (const nested in exports) if (nested.length <= 3) {
-                            if (exports[nested] === window) {
-                                foundWindow = true;
-                            }
+                let shouldMakeNonEnumerable = false;
+
+                nonEnumerableChecking: {
+                    // There are (at the time of writing) 11 modules exporting the window,
+                    // and also modules exporting DOMTokenList, which breaks webpack finding
+                    // Make these non enumerable to improve search performance and avoid erros
+                    if (exports === window || exports[Symbol.toStringTag] === "DOMTokenList") {
+                        shouldMakeNonEnumerable = true;
+                        break nonEnumerableChecking;
+                    }
+
+                    if (typeof exports !== "object") {
```

Comment:
just in case (typeof null is object)

```suggestion
                    if (typeof exports !== "object" || exports === null) {
```